infra/build/functions/request_build.py

```diff
@@ -23,11 +23,26 @@
 
 import build_lib
 import build_project
+from datastore_entities import BuildsHistory
 from datastore_entities import Project
 
 BASE_PROJECT = 'oss-fuzz-base'
 
 
+def update_build_history(project_name, build_id, tag):
+  """Update build history of project."""
+  build_history = BuildsHistory.query(BuildsHistory.project==project_name, BuildsHistory.tag==tag)
+  project = build_history.get()
```

Comment:
"project" is not a the right variable name here. Use "build_history", and rename the query to "build_history_query".